task3/frontpage.css

```diff
@@ -0,0 +1,253 @@
+head{
```

Comment:
The head element can not contain styles (it contains the meta data of the page like related resources, title, information for search bots)